op-guide/migration.md

```diff
@@ -256,12 +256,45 @@ log-level = "info"
 
 server-id = 101
 
-# the file path for meta
 meta = "./syncer.meta"
-worker-count = 1
-batch = 1
-
-pprof-addr = ":10081"
+worker-count = 16
+batch = 10
+
+status-addr = ":10081"
+
+skip-sqls = ["ALTER USER", "CREATE USER"]
+
+##replicate-do-db priority over replicate-do-table if have same db name
+##and we support regex expression , start with '~' declare use regex expression.
+#
+#replicate-do-db = ["~^b.*","s1"]
+#[[replicate-do-table]]
+#db-name ="test"
+#tbl-name = "log"
+
+#[[replicate-do-table]]
+#db-name ="test"
+#tbl-name = "~^a.*"
+
+# skip prefix mathched sqls
+# skip-sqls = ["^ALTER\\s+USER", "^CREATE\\s+USER"]
+
+// 1. asterisk character (*, also called "star") matches zero or more characters,
+//    for example, doc* matches doc and document but not dodo;
+//    asterisk character must be in the end of wildcard word,
+//    and there is only one asterisk in one wildcard word
+// 2. the question mark ? matches exactly one character
```

Comment:
The format of this comment seems odd.  It is displayed in red. Could you please take a look?